From 9ee739612e9ae926ee985bc79adafb43ab42c872 Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Mon, 3 Mar 2025 17:18:54 +0100
Subject: [PATCH] add max_idle_secs to bazel startup args for win

---
 setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index 8505aa0..d8d7630 100644
--- a/setup.py
+++ b/setup.py
@@ -157,6 +157,8 @@ class BuildBazelExtension(build_ext.build_ext):
       # Required to build protobuf. See
       # https://github.com/protocolbuffers/protobuf/issues/12947
       bazel_startup_args += [r'--output_base=C:\O']
+      # Required to shutdown Bazel and make it ready for the next build
+      bazel_startup_args += [r'--max_idle_secs=1']
       # Needed by pybind11_bazel.
       os.environ['PYTHON_BIN_PATH'] = sys.executable
       os.environ['PYTHON_LIB_PATH'] = os.path.join(sys.exec_prefix, 'lib')
-- 
2.39.1

